sdf/1.11/mimic.sdf

```diff
@@ -0,0 +1,37 @@
+<!-- Mimic -->
+<element name="mimic" required="0">
+  <description>Specifies a linear equality constraint between the position of two joint
+    axes. One joint axis is labeled as the leader and the other as the follower.
+    The joint axis containing a mimic tag is the follower, and the leader
+    is specified by the joint and axis attributes.
+    The multiplier, offset, and reference parameters determine the linear relationship
+    according to the following equation:
+    follower_position = multiplier * (leader_position - reference) + offset.
+    Note that the multiplier and offset parameters match the parameters of
+    the URDF mimic tag if the reference parameter is 0.
+  </description>
+  <attribute name="joint" type="string" default="" required="1">
+    <description>Name of the joint containing the leader axis, i.e. the axis to be mimicked</description>
+  </attribute>
+  <attribute name="axis" type="string" default="axis" required="0">
```

Comment:
Should this be required? Not sure, but all the other attributes are

Reply:
When you say "all the other attributes are", it sounds like you're talking about attributes in other parts of the spec, because mimic has only one other attribute (`//mimic/@joint`), which admittedly is required. There are non-required attributes in other parts of the spec, though, such as [//pose/@relative_to](http://sdformat.org/spec?ver=1.10&elem=joint#pose_relative_to) and [//physics/@name](http://sdformat.org/spec?ver=1.10&elem=physics#physics_name).

My understanding of marking an attribute as non-required is that it won't be auto-populated with the default value into a file where it isn't present. With that in mind, I marked `//mimic/@axis` as non-required to reduce noise in the files, though `//mimic/@joint` is required since the user must specify it.

Reply:
Oh, sorry, I meant just the attributes in `//mimic` and I see now `multiplier`, `offset` and `reference` are elements. I don't have a strong opinion. Just wanted to make sure it was an intentional choice.